src/main.js

```diff
@@ -5,6 +5,7 @@
 import Vue from 'vue'
 import App from './App'
 import router from './router'
+import FullCalendar from 'vue-full-calendar';
```

Comment:
have you added this in `package.json` ?

Reply:
Yes, this has been included in package.json for accounting,